pyiron_base/project/project_browser.py

```diff
@@ -0,0 +1,337 @@
+import os
```

Comment:
Add encoding and copyright to head of file:
```python
# coding: utf-8
# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
# Distributed under the terms of "New BSD License", see the LICENSE file.
```

Reply:
Thanks, forgot that.